### ISSUE TITLE
Escape sequence is sent to the remote side, which should be avoid

### ISSUE THREAD

**Issue by neo954** [xcat2/goconserver#2]
This bug is against `goconserver` `Version: 0.1, BuildTime: 2017-11-02T06:57:35-0400 Commit: 7e9278e88eb4f3035707b41f0a717fede7291498` running on a ppc64le Red Hat 7.4 Linux environment.

**The recreation steps**

Create a node definition in `congo`. And then connect to the newly created node. BTW, the node, `c910f03c01p09` is just another common Linux node, which runs Red Hat 7.4.

```
# congo create c910f03c01p09 driver=ssh ondemand=false \
> --params user=root,host=10.3.1.9,port=22,password=a_password
# congo console c910f03c01p09
goconserver(2017-11-07 23:47:16): Hello 127.0.0.1:57278, welcome to the session of c910f03c01p09

[root@c910f03c01p09 ~]# showkey -a

Press any keys - Ctrl-D will terminate this program
                                <<<-- PRESS `Ctrl-E', `c', AND `.' HERE 
^E 	  5 0005 0x05
c 	 99 0143 0x63
```

**What actually happened**

It seems, the escape sequence, `Ctrl-E`, `c` was sent to the remote side through ssh session.

**What is expected**

Here is the expect behavior, in this way, the escape sequence will not send to the remote side.

* While `Ctrl-E` is pressed by the user, hold it in the buffer, do not sent it to the remote side.
* If the next input character is not `c`, sent `Ctrl-E` and this input character to the remote side.
* Or, If the input character is `c`, hold it in the buffer as well.
* If the next input character is not `.`, sent `Ctrl-E`, `c` and this input character to the remote side.
* Or, If the input character is `.`, clear the buffer, disconnect the session.

**Comment by chenglch** [comment]
The fix for this issue has been included in the commit 906cd810f009dd1b0b22a75ea0c4efc1cbb2e840.

**Comment by neo954** [comment]
This issue was fixed long time ago. I will close this one.
